core/controllers/library.py

```diff
@@ -177,12 +196,22 @@ class LibraryGroupIndexHandler(base.BaseHandler):
     """Provides data for categories such as top rated and recently published."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'group_name': {
```

Comment:
Please add the choices.

Reply:
Here also it is raising PageNotFoundException if the group_name conatins other than 'recently-published' or 'top-rated'. 
Please take a look.